### ISSUE TITLE
Add example of PyMC Jax and Black Jax

### ISSUE THREAD

**Issue by canyon289** [bambinos/bambi#525]
creating per conversation with @zwelitunyiswa in this repository https://github.com/bambinos/bambi/issues/522

To make a PR you'll need to
- [ ] Use git to clone this repository
- [ ] Follow the setup instructions
- [ ] Add a documentation and compile it locally to check it worked
- [ ] Make a PR on github

**Comment by zwelitunyiswa** [comment]
@canyon289  Is this still necessary given that Jax has been implemented with #522  ?

**Comment by aloctavodia** [comment]
Yes, we still need an example. @zwelitunyiswa do you want to do it?

**Comment by zwelitunyiswa** [comment]
Yes, was just checking . Work on it tomorrow.

**Comment by speco29** [comment]
I can add those examples, would you mind telling me where to add these?
I'm a bit confused.

**Comment by tomicapretto** [comment]
We already have this https://bambinos.github.io/bambi/notebooks/alternative_samplers.html
